<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMed</class>
 <widget class="QDialog" name="AddMed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new medicine</string>
  </property>
  <property name="styleSheet">
   <string notr="true">AddMed {
	background-color: rgb(255, 255, 255);
}


QPushButton {
 background-color: red;
 color: white;
 padding: 10px 20px;
 text-align: center;
 font-size: 16px;
 margin: 4px 2px;
 border-radius: 4;
 font: 57 8pt &quot;TT Norms Medium&quot;;
}

QPushButton:hover {
     background-color: #FF5757;
}

QPushButton:pressed {
  background-color: #3e8e41;
}

QTextBrowser {
	background-color: white;
	color: red;
	border: none;
   border-radius: 4;
   font: 57 8pt &quot;TT Norms Light&quot;;
}

QLabel {
 font: 57 8pt &quot;TT Norms Medium&quot;;
text-align: center;
color: red;
}

QComboBox {
background-color: rgb(255, 255, 255);
 font: 57 8pt &quot;TT Norms Medium&quot;;
border: none;
}

QSpinBox {
background-color: rgb(255, 255, 255);
 font: 57 8pt &quot;TT Norms Medium&quot;;
border: none;
}

QLineEdit {
 font: 57 8pt &quot;TT Norms Medium&quot;;
}
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name*:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFreq"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>How long (days/weeks)*:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxTime"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Comment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditComment"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
